test/system/annotator_page_test.rb

```diff
@@ -0,0 +1,25 @@
+require "application_system_test_case"
+require 'webmock/minitest'
+
+class AnnotatorPageTest < ApplicationSystemTestCase
+    def setup
+        WebMock.disable!
+        @apikey = LinkedData::Client.settings.apikey
+        @host = LinkedData::Client.settings.rest_url
+        @annotator_api = "https://services.stageportal.lirmm.fr/annotator"
+        @annotator_text_area = "div#annotator_text_area"
```

Comment:
you can find elements using css selector, without creating each time an id just for the test. here, for example, it would be `.annotator-page-text-area > textarea`

Reply:
Yes, I did this only for the main text area

Reply:
the change request here; is to remove  `#annotator_text_area` from the view

Reply:
ok